src/main/java/life/qbic/portal/portlet/OMEROClientPortlet.java

```diff
@@ -450,11 +450,12 @@ private Button metadataButton(long imageId) {
             }
 
         } catch (Exception e) {
-            LOG.error("Could not retrieve metadata for image:" + imageId);
-            LOG.debug(e);
+
+            return metadataButton;
 
         }
         metadataButton.addClickListener(clickEvent -> {
+            try {
             Window metadataSubWindow = new Window("Metadata Sub-Window");
```

Comment:
indent

Reply:
Sorry i have no idea what happened to my IDE format i reverted the commit!